src/components/item-blog.tsx

```diff
@@ -16,19 +16,17 @@ export const ItemBlog: React.FC<{ data: ItemBlogProps}> = ({ data }) => {
             <div className={`transition-all duration-300 hover:shadow-2xl shadow ${focused && 'focused'}`}>
                 <Link to={data.fields.slug} title={data.frontmatter.title} onFocus={() => changeFocused(true)} onBlur={() => changeFocused(false)}>
                     <div className="image">
-                        <Img
-                            fluid={data.frontmatter.image.childImageSharp.fluid}
-                            alt={data.frontmatter.title}
-                            className="w-full"
-                        />
+                        {data.frontmatter.image.publicURL.endsWith('.svg') ?
+                        <img src={data.frontmatter.image.publicURL} alt={data.frontmatter.title} className="w-full" /> :
+                        <Img fluid={data.frontmatter.image.childImageSharp?.fluid} alt={data.frontmatter.title} className="w-full" />}
```

Comment:
Might be worth adding prettier to the project as a next task. 
https://prettier.io/

Reply:
Is that a replacement for the Prism plugin we're using now?

Reply:
I would standardize our code formatting across all the files to make this indentation more normalized.